RELEASES.md

```diff
@@ -1,3 +1,120 @@
+Version 1.50.0 (2021-02-11)
+============================
+
+Language
+-----------------------
+- [You can now use `const` values for `T` in `[T; N]` array expressions.][79270]
+- [Assignments to `ManuallyDrop<T>` union fields are now considered safe.][78068]
+
+Compiler
+-----------------------
+- [Added tier 3\* support for the `armv5te-unknown-linux-uclibceabi` target.][78142]
+- [Added tier 3 support for the `aarch64_apple_ios_macabi` target.][77484]
+- [The `x86_64-unknown-freebsd` is now built with the full toolset.][79484]
+
+\* Refer to Rust's [platform support page][forge-platform-support] for more
+information on Rust's tiered platform support.
+
+Libraries
+-----------------------
+
+- [`proc_macro::Punct` now implements `PartialEq<char>`.][78636]
+- [On Unix platforms, the `std::fs::File` type now has a "niche" of `-1`.][74699]
+  This value cannot be a valid file descriptor, and now means `Option<File>` takes
+  up the same amount of space as `File`.
+
+Stabilized APIs
+---------------
+
+- [`UnsafeCell::get_mut`]
+- [`bool::then`]
+- [`RefCell::take`]
+- [`btree_map::Entry::or_insert_with_key`]
+- [`hash_map::Entry::or_insert_with_key`]
+- [`slice::fill`]
+
+The following previously stable methods are now `const`.
+
+- [`IpAddr::is_ipv4`]
+- [`IpAddr::is_ipv6`]
+- [`Layout::size`]
+- [`Layout::align`]
+- [`Layout::from_size_align`]
+- `pow` for all integer types.
+- `checked_pow` for all integer types.
+- `saturating_pow` for all integer types.
+- `wrapping_pow` for all integer types.
+- `next_power_of_two` for all unsigned integer types.
+- `checked_power_of_two` for all unsigned integer types.
+
+Cargo
+-----------------------
+
+- [Added the `[build.rustc-workspace-wrapper]` option.][cargo/8976]
+  This option sets the `rustc` wrapper to use to build workspace members.
+- [`cargo:rerun-if-changed` will now, if provided a directory, scan the entire
+  contents of that directory for changes.][cargo/8973]
+- [Periods (`.`) are now allowed in feature names.][cargo/8932]
+- [Added the `--workspace` flag to the `cargo update` command.][cargo/8725]
+
+Misc
+----
+
+- [The search results tab and help button are focusable with keyboard in rustdoc.][79896]
```

Comment:
```suggestion
- [The search results Tab and Help buttons are focusable with keyboard in rustdoc.][79896]
```

Reply:
I'm not sure it's correct capitalise here?

Reply:
Ah, nevermind, I had read the sentence as "... (tab and help) buttons", which is why I added the plural. Perhaps:
```suggestion
- [The search results tab and the help button are focusable with keyboard in rustdoc.][79896]
```